components/infobox/wikis/overwatch/infobox_league_custom.lua

```diff
@@ -0,0 +1,249 @@
+---
+-- @Liquipedia
+-- wiki=overwatch
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+local PageLink = require('Module:Page')
+local PrizePoolCurrency = require('Module:Prize pool currency')
+
+local _TODAY = os.date('%Y-%m-%d', os.time())
+
+local _args
+local _league
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _GAMES = {
+	overwatch = 'Overwatch',
+	overwatch2 = 'Overwatch 2'
+}
+
+local _BLIZZARD_TIERS = {
+	owl = 'Overwatch League',
+	owc = 'Overwatch Contenders',
+}
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+	_args = _league.args
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.getWikiCategories = CustomLeague.getWikiCategories
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {(args.team_number or '') .. (args.team_slots and ('/' .. args.team_slots) or '')}
```

Comment:
if `team_slots` is set but `team_number` is not set the display will look off
also i did not find any usage of `team_slots` via a search on overwatch wiki